src/coreclr/ilasm/writer.cpp

```diff
@@ -51,10 +52,45 @@ HRESULT Assembler::InitMetaData()
     if(FAILED(hr = m_pEmitter->QueryInterface(IID_IMetaDataImport2, (void**)&m_pImporter)))
         goto exit;
 
+    if (m_fDeterministic)
+    {
+        //
+        // In deterministic mode, the MVID will need to be stabilized for the metadata scope that
+        // was created above, and the ChangeMvid service that makes this possible is only available
+        // on the IMDInternalEmit interface.
+        //
+        // When the CLSID_CorMetaDataDispenser instance above has activated against a current
+        // clr.dll (which is the only supported configuration for the determinism feature), it is
```

Comment:
ilasm in .NET Core has statically linked copy of metadata implementation. This comment that talks about clr.dll is not relevant here. Was it copied from .NET Framework port?

Reply:
BTW: I do not think you would be able to use IMDInternalEmit interface for the .NET Framework port. It would create versioning problems. I think we will need a proper new interface defined for .NET Framework port.

Reply:
> Was it copied from .NET Framework port?

I believe so -- this comment is from the initial commit from the parent PR.